_strategies/agile-development.md

```diff
@@ -10,36 +10,36 @@ image-credit: https://www.flickr.com/photos/hernanpc/
+To help this process, we pair identifying goals at a higher level (an “epic” in agile terminology) with an ongoing user research process. These epics describe the user need, not the solution.  For example, a user story might be: “As a caseworker, I want to see all my current cases and next steps so I know what I need to do next, and when I need to do it.” We wouldn't write: “The system shall display the next step next to each case in a grid on the main screen.” Through the user research and usability testing throughout the life of the product development, these epics are turned into a set of increasingly-specific user stories. Each story should be designed, researched, developed and tested within a single sprint. We validate each implemented story with usability testing with real end users, and use the feedback from that work to generate addtional backlog user stories.
 
-There are three leading causes why IT projects fail: a lack of reliable input from users, incomplete requirements and specifications, and changes in the requirements and specifications. The Standish Group, an IT research firm, publishes an annual report on IT projects, in which they point out that these causes of failure are about the impossibility of “knowing” requirements up front.
+ We prioritize user stories continuously and do not focus on what is in the development backlog beyond what can be done in a month in the future.
 
-#### Agile development requires a shift in how you understand the costs of software development.
+### Agile development requires a shift in how you understand the costs of software development.
 
-The traditional “waterfall” method of software development assumes that software can be built once for a large, one-time investment. In practice, this is untrue, because requirements can rarely be fully known and change constantly. Users' needs tend to arise through research and testing, and development teams must discover appropriate solutions by building iteratively and interacting with users regularly.
+The traditional “waterfall” method of software development assumes that software can be built once for a large, one-time investment. In practice, this is untrue, because requirements can rarely be fully known and change constantly. The product team's understanding of users' needs arise through research and testing which deepens but shortens your teams forecasting of costs..
 
```

Comment:
Suggest replacing "constantly" with "regularly," "often," or "frequently." Extra period at end of last sentence.

I don't really understand what "deepens but shortens your teams forecasting of costs" means.